### DISCUSSION TITLE
Add right to left text for Hebrew language

### DISCUSSION THREAD

**Discussion by EytanT** [SalesforceCommerceCloud/sfcc-ci#465] [Team Posts]
Hi guys,
I want to add the right to left text display for languages like hebrew.
How can i do this?

Thanks to all

**Comment by gpenesfdc** [comment]
Hi EytanT, I have a site working with the he_IL locale. you can check it here:
https://gpene-inside-eu02-dw.demandware.net/s/RefArchGlobal/he_IL/home
You need to do several thing:
1) define the he_IL locale in Administration >  Global Preferences >  Locales
2) activate the he_IL locale for your site (and define it as default if needed)
3) activate the ILS currency
4) extend the countries.json file in SFRA with the following:
, {
    "id": "he_IL",
    "currencyCode": "ILS"
}
I have also started to change the CSS to be right to left.
Greg

**Comment by EytanT** [comment]
Thanks

**Comment by deivan** [comment]
Unfortunately, your question is very common, let you try ask some more concretion?..
First of all try add attribute `dir="rtl"` to your component tags, for example
`<body dir="rtl">`
